pkg/api/converterfromv20180930preview.go

```diff
@@ -32,6 +32,12 @@ func ConvertFromV20180930preview(oc *v20180930preview.OpenShiftManagedCluster) *
 			FQDN:              oc.Properties.FQDN,
 		}
 
+		// if router profile is not set on the external api, default to minimum
+		if len(oc.Properties.RouterProfiles) == 0 {
+			oc.Properties.RouterProfiles = make([]v20180930preview.RouterProfile, 1)
+			oc.Properties.RouterProfiles[0].Name = "default"
```

Comment:
I think we should not enforce name "default" to router profiles. 
Now if I provide just bare minimum:
```
  routerProfiles:
  - publicSubdomain: $RESOURCEGROUP.$DNS_DOMAIN
```
 will get validation error:
```
[invalid properties.routerProfiles[""], invalid properties.routerProfiles[""].name "", invalid properties.routerProfiles["default"]]
```
Which implies now cusomter will need to know this "default" name to be default. Less customers need to worry, better. 

We could either:
1. Default first router to name `default` always
2. Remove validation on name == `default`.

Reply:
What was the idea behind `default` as a mandatory name?